README.md

```
@@ -1,19 +1,33 @@
 
-```sh
-aws --endpoint-url=http://localhost:9324 sqs list-queues
+```bash
+# Start ElasticMQ and the GUI on ports 9324, 9325, and 8080
+docker compose up -d
+
+# Tail logs if you want to watch the services
+docker compose logs -f
 ```
+
+Once the containers are healthy, open http://localhost:8080/queues to browse queues and send messages. The ElasticMQ service automatically seeds the configuration from `elasticmq.conf`, and the GUI container inherits dummy credentials plus `AWS_SQS_ENDPOINT=http://elasticmq:9324` so everything works out of the box.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)